building nd4j fails on jetson nano board arm64 armv8-a

#### Issue Description

I recently started trying to build deeplearning4j on the Jetson Nano board.  I was able to compile libnd4j, however when i then try to go and build nd4j, it fails due to missing dependencies for  "mkl-dnn"  ..  

So, I began building mkl-dnn from javacpp-presets natively on the jetson nano, unfortuantely this fails because mkl-dnn depends on x86 specific included header files  ( cpuid.h for example that is not available on the arm platform.)

so..  The questions is, should i skip trying to do the native builds and only do the cuda builds for that platform?   I tried this by removing the references to nd4j-native / nd4j-native-platform backend impls, ..  the build got further, however then crashed in nd4j-grpc  due to another dependency on dnn-mkl.  


```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for nd4j 1.0.0-beta3:
[INFO]
[INFO] nd4j ............................................... SUCCESS [  9.094 s]
[INFO] nd4j-shade ......................................... SUCCESS [  0.674 s]
[INFO] jackson ............................................ SUCCESS [ 53.685 s]
[INFO] nd4j-common ........................................ SUCCESS [ 18.707 s]
[INFO] nd4j-context ....................................... SUCCESS [  3.935 s]
[INFO] nd4j-buffer ........................................ SUCCESS [  6.378 s]
[INFO] nd4j-backends ...................................... SUCCESS [  0.414 s]
[INFO] nd4j-api-parent .................................... SUCCESS [  0.469 s]
[INFO] nd4j-api ........................................... SUCCESS [03:30 min]
[INFO] nd4j-jdbc .......................................... SUCCESS [  0.396 s]
[INFO] nd4j-jdbc-api ...................................... SUCCESS [  3.456 s]
[INFO] nd4j-jdbc-mysql .................................... SUCCESS [  3.521 s]
[INFO] nd4j-jdbc-hsql ..................................... SUCCESS [  4.002 s]
[INFO] nd4j-instrumentation ............................... SUCCESS [  4.652 s]
[INFO] nd4j-serde ......................................... SUCCESS [  0.299 s]
[INFO] nd4j-aeron ......................................... SUCCESS [  6.922 s]
[INFO] nd4j-base64 ........................................ SUCCESS [  2.952 s]
[INFO] nd4j-jackson ....................................... SUCCESS [  3.695 s]
[INFO] nd4j-kryo .......................................... SUCCESS [  6.883 s]
[INFO] nd4j-camel-routes .................................. SUCCESS [  0.282 s]
[INFO] nd4j-kafka ......................................... SUCCESS [  3.437 s]
[INFO] nd4j-gson .......................................... SUCCESS [  3.111 s]
[INFO] nd4j-arrow ......................................... SUCCESS [  3.499 s]
[INFO] nd4j-grpc .......................................... FAILURE [01:13 min]
[INFO] nd4j-native-api .................................... SKIPPED
[INFO] nd4j-tests ......................................... SKIPPED
[INFO] nd4j-backend-impls ................................. SKIPPED
[INFO] nd4j-cuda .......................................... SKIPPED
[INFO] nd4j-cuda-platform ................................. SKIPPED
[INFO] nd4j-tensorflow .................................... SKIPPED
[INFO] nd4j-tests-tensorflow .............................. SKIPPED
[INFO] nd4j-parameter-server-parent ....................... SKIPPED
[INFO] nd4j-parameter-server-model ........................ SKIPPED
[INFO] nd4j-parameter-server .............................. SKIPPED
[INFO] nd4j-parameter-server-status ....................... SKIPPED
[INFO] nd4j-parameter-server-client ....................... SKIPPED
[INFO] nd4j-parameter-server-rocksdb-storage .............. SKIPPED
[INFO] nd4j-parameter-server-node ......................... SKIPPED
[INFO] nd4j-uberjar ....................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  07:07 min
[INFO] Finished at: 2019-05-07T23:54:12-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project nd4j-grpc: Could not resolve dependencies for project org.nd4j:nd4j-grpc:jar:1.0.0-beta3: The following artifacts could not be resolved: org.nd4j:nd4j-native:jar:linux-arm64:1.0.0-beta3, org.bytedeco.javacpp-presets:mkl:jar:linux-arm64:2019.0-1.4.3, org.bytedeco.javacpp-presets:mkl-dnn:jar:linux-arm64:0.16-1.4.3: Could not find artifact org.nd4j:nd4j-native:jar:linux-arm64:1.0.0-beta3 in maven-restlet (http://maven.restlet.org) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
```

--- a/libnd4j/blas/CMakeLists.txt
+++ b/libnd4j/blas/CMakeLists.txt
@@ -44,10 +44,6 @@ if(WIN32)
     SET(CMAKE_CXX_RESPONSE_FILE_LINK_FLAG "@")
 
     SET(CMAKE_NINJA_FORCE_RESPONSE_FILE 1 CACHE INTERNAL "")
-else()
-    if (CUDA_BLAS)
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mcmodel=medium")
-    endif()
 endif()
 
 

--- a/libnd4j/buildnativeoperations.sh
+++ b/libnd4j/buildnativeoperations.sh
@@ -169,6 +169,12 @@ case "$OS" in
     fi
     ;;
 
+    linux-arm64)
+    if [ -z "$ARCH" ]; then
+        ARCH="armv8-a"
+    fi
+    ;;
+
     android-arm)
     if [ -z "$ARCH" ]; then
         ARCH="armv7-a"

--- a/nd4j/nd4j-backends/nd4j-backend-impls/pom.xml
+++ b/nd4j/nd4j-backends/nd4j-backend-impls/pom.xml
@@ -421,6 +421,20 @@
           </properties>
         </profile>
         <profile>
+          <id>linux-arm64-default</id>
+          <activation>
+            <property>
+              <name>javacpp.platform</name>
+              <value>linux-arm64</value>
+            </property>
+          </activation>
+          <properties>
+            <javacpp.platform.properties>linux-arm64</javacpp.platform.properties>
+            <javacpp.platform.compiler>aarch64-linux-gnu-g++</javacpp.platform.compiler>
+            <dependency.platform2></dependency.platform2>
+          </properties>
+        </profile>
+        <profile>
           <id>linux-ppc64le-default</id>
           <activation>
             <property>

